src/main/java/com/gorlah/kappabot/integration/discord/DiscordBot.java

```diff
@@ -19,74 +20,75 @@
 import java.util.Map;
 
 @Component
+@RequiredArgsConstructor
 public class DiscordBot extends ListenerAdapter {
-    
-    @Value("${discord.token}")
-    private String discordToken;
-    
+
     @Value("${discord.command.prefix}")
     private String commandPrefix;
-    
-    private JDA bot;
-    
+
     private final CommandProcessor commandProcessor;
-    
-    public DiscordBot(CommandProcessor commandProcessor) {
-        this.commandProcessor = commandProcessor;
-    }
-    
+    private final DiscordIntegration discordIntegration;
+
+    private JDA bot;
+
     @PostConstruct
-    private void initialize() throws LoginException {
-        bot = new JDABuilder(discordToken)
-                .addEventListeners(this)
-                .setActivity(Activity.playing(commandPrefix + " help | I'm back. \uD83D\uDE0F"))
-                .build();
+    private void initialize()
+            throws LoginException {
+        if (discordIntegration.isEnabled()) {
+            bot = new JDABuilder(discordIntegration.getToken())
+                    .addEventListeners(this)
+                    .setActivity(Activity.playing(commandPrefix + " help | I'm back. \uD83D\uDE0F"))
+                    .build();
+        }
     }
-    
+
     @Override
     public void onMessageReceived(MessageReceivedEvent event) {
         if (event.getAuthor().isBot()) {
             return;
         }
-        
+
         String message = event.getMessage().getContentRaw();
         StringBuilder response = new StringBuilder();
-        
+
         if (SlashdotParser.find(message)) {
             ArrayList<String> titles = SlashdotParser.getTitles(message);
-    
+
             for (int i = 0; i < titles.size(); i++) {
                 if (i > 0) {
                     response.append("\n");
                 }
-                
+
                 response.append("> ").append(titles.get(i));
             }
         } else {
             if (message.length() < commandPrefix.length()
-                    || !commandPrefix.equalsIgnoreCase(message.substring(0, commandPrefix.length()))) {
+                    || !commandPrefix.equalsIgnoreCase(message.substring(
+                    0,
+                    commandPrefix.length()))) {
                 return;
             }
-            
-            Command command = new Command(commandPrefix, message, event.getAuthor().getName());
-    
+
+            Command command =
+                    new Command(commandPrefix, message, event.getAuthor().getName());
```

Comment:
I don't think this line is long enough to warrant a break.  Was this intended?  Is this that the project code style can be updated for?

Reply:
I don't know why this one got wrapped, I didn't specifically do it.

Reply:
Weird.  I'll double check the code style.  Thanks!